drivers/i2c/slave/eeprom_slave.c

```diff
@@ -212,7 +212,7 @@ static u8_t i2c_eeprom_slave_0_buffer[(DT_INST_0_ATMEL_AT24_SIZE)];
 static const struct i2c_eeprom_slave_config i2c_eeprom_slave_0_cfg = {
 	.controller_dev_name = DT_INST_0_ATMEL_AT24_BUS_NAME,
 	.address = DT_INST_0_ATMEL_AT24_BASE_ADDRESS,
-	.buffer_size = (DT_INST_0_ATMEL_AT24_SIZE * 1024),
+	.buffer_size = (DT_INST_0_ATMEL_AT24_SIZE),
```

Comment:
The same fix is needed for instance 1. Would you mind fixing that too?

Reply:
Done